examples/php/helloGet.php

```diff
@@ -0,0 +1,6 @@
+<?php
+
+function foo() {
+  print "hello world";
+}
+
```

Comment:
we should have another example in which the data given in the request is used somehow. 

Also you should add in the Makefile of the `examples/` folder  task to deploy and verify these examples. Note that there should be tasks there to verify that setting a custom port and timeouts works as expected.

Those tests will be executed in travis if you add them in the file `./tests/integration-tests.bats`

Reply:
Ok thanks @andresmgot, i'm going to add it in the next commit.

Reply:
@andresmgot added a test with data and added to integration-tests.bats